teamA/lib/Views/dashboard.dart

```diff
@@ -0,0 +1,279 @@
+            // Large screen (desktop or large tablet)
+            return _buildDesktopLayout(context);
+          } else {
+            // Small screen (mobile)
+            return _buildMobileLayout(context);
+          }
+        },
+      ),
+    );
+  }
+
+  // Desktop layout
+  Widget _buildDesktopLayout(BuildContext context) {
+    return Center(
+      child: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Text(
+              'Teacher Dashboard',
+              style: TextStyle(
+                fontSize: 24,
+                fontWeight: FontWeight.normal,
+                color: Theme.of(context).colorScheme.onBackground,
+              ),
+            ),
+            SizedBox(height: 40), // Space between the title and buttons
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Column(
+                  children: [
+                    Text(
+                      'Teacher can view available courses here.',
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: Theme.of(context).colorScheme.onBackground,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                    _buildDashboardButton(
+                        context, 'Courses', 150), // Smaller button
+                  ],
+                ),
+                SizedBox(width: 20),
+                Column(
+                  children: [
+                    Text(
+                      'Teacher creates/views assessments.',
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: Theme.of(context).colorScheme.onBackground,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                    _buildDashboardButton(
+                        context, 'Assessments', 180), // Larger middle button
+                  ],
+                ),
+                SizedBox(width: 20),
+                Column(
+                  children: [
+                    Text(
+                      'Teacher can view, grade, and create essays.',
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: Theme.of(context).colorScheme.onBackground,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                    _buildDashboardButton(
+                        context, 'Essays', 150), // Smaller button
+                  ],
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  // Mobile layout
+  Widget _buildMobileLayout(BuildContext context) {
+    return Center(
+      child: SingleChildScrollView(
```

Comment:
Personal preference is to put the Center inside the Scroll View so that the scroll bar appears on the edge of the window.

Reply:
resolved it.